### DISCUSSION TITLE
New release with pfx support

### DISCUSSION THREAD

**Discussion by tripleclones** [go-acme/lego#1564] [Q&A] [answered]
### Welcome

- [X] Yes, I've searched similar issues on GitHub and didn't find any.

### How do you use lego?

Binary

### Detailed Description

Would it be possible to get a new release with pfx support? 

I've being try to build a windows binary from source myself but failing.

**Comment by tripleclones** [comment, marked as answer]
Just to say I have managed to build a version and have tested it on Windows and Server 2019 without issue

**Reply by ldez** [reply]
https://github.com/go-acme/lego/releases/tag/v4.6.0
